web/client/examples/rasterstyler/assets/css/home.css

```diff
@@ -0,0 +1,100 @@
+html, body, #container, .home {
```

Comment:
I think all these css are not needed, just include one css with the needed stuff